### DISCUSSION TITLE
How do I translalate to another language than English?

### DISCUSSION THREAD

**Discussion by MohammedMehdiTBER** [linto-ai/whisper-timestamped#119] [Q&A]
I want to know how to change the script to another language and if I can choose from many languages to process at once along with the transcribe option.

**Comment by Jeronymous** [comment]
Whisper models were not trained to other translation tasks that <some language> to English.
So translating to another language would require re-training / finetuning Whisper models.
However, we noticed that even without retraining, if you ask Whisper to transcribe in a given language that is not English, on an audio that is in English, it can translate to this language. It sometimes works for French and Spanish. But as Whisper was not trained to do this, this behavior is not reliable at all.
A possible pipeline is : ask Whisper to translate to English (if the audio is not in English) then use another translation model to translate to your target language. So using English as a pivot language.
